### PR TITLE
Add temporary backup cookie handling

### PR DESCRIPTION
**Pull request by MrCreosote**
Accepts the temporary kbase_session_backup cookie.

Also some line length cleanup, updating workspace error messages, and a trivial code simplification

### CODE REVIEW

File: lib/src/htmlfilesetserv/HTMLFileSetHTTPServer.java [hunk 1]
```diff
@@ -687,19 +641,22 @@ private AuthToken getToken(final HttpServletRequest request)
...
-						!c.getValue().isEmpty()) {
-					return auth.validateToken(
-							unmungeCookiePerShane(c.getValue()));
+				if (c.getName().equals(TOKEN_COOKIE_NAME) && !c.getValue().isEmpty()) {
+					return auth.validateToken(unmungeCookiePerShane(c.getValue()));
 				}
 			}
 		}
+		// hacky hack hack - remove later when a final solution for cookies is found
+		for (final Cookie c: request.getCookies()) {
```
Comment (rsutormin) [inline lib/src/htmlfilesetserv/HTMLFileSetHTTPServer.java]: This second TOKEN_COOKIE2_NAME case should go inside `if (request.getCookies() != null) {}` block (one code line up).

Comment (MrCreosote) [inline lib/src/htmlfilesetserv/HTMLFileSetHTTPServer.java]: fixed

**Comment by MrCreosote** [comment]
@rsutormin can you re-review?

**Comment by scanon** [comment]
Looks good to me.

**Pull request merged**